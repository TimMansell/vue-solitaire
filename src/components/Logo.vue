<template>
  <div class="logo">
    <img class="logo__image" :src="logo" />
  </div>
</template>

<script>
import logo from '@/assets/logo.svg';

export default {
  name: 'Logo',
  data() {
    return {
      logo,
    };
  },
};
</script>

<style lang="scss">
.logo {
  display: flex;
  justify-content: center;
  width: 100%;
  margin-bottom: var(--vr);

  &__image {
    width: 100px;

    @media (min-width: $bp-sm) {
      width: 175px;
    }
  }
}
</style>
