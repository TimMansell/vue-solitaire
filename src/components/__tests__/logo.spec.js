import { shallowMount } from '@vue/test-utils';
import Logo from '@/components/Logo.vue';

describe('Logo.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Logo);

    expect(wrapper).toMatchSnapshot();
  });
});
