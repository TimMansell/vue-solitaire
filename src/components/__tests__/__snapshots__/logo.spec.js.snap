// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo.vue matches snapshot 1`] = `<div class="logo"><img src="" class="logo__image"></div>`;
